client/scripts/components/TorrentList/Torrent.js

```diff
@@ -147,6 +145,51 @@ export default class Torrent extends React.Component {
       );
     }
 
+    let tertiaryDetails = [
+      <li className="torrent__details--completed">
```

Comment:
We need to add a `key` attribute to each of these `<li>` elements because they are children of an array. A simple string will be fine, it just needs to be unique for each item in the array. Same for the pushed element on line 185.